.gitignore

```diff
@@ -0,0 +1,2 @@
+*.swp
+vendor
```

Comment:
^^ and also have a separate way of tracking those differences so you know what to pull. But doesn't make sense to the be in the git repository. haha all this because `glide mirror` is totally messed, never been able to get it to work - have you?